llvm/lib/Target/AMDGPU/SIRegisterInfo.cpp

```diff
@@ -797,6 +797,23 @@ int64_t SIRegisterInfo::getScratchInstrOffset(const MachineInstr *MI) const {
 
 int64_t SIRegisterInfo::getFrameIndexInstrOffset(const MachineInstr *MI,
                                                  int Idx) const {
+  switch (MI->getOpcode()) {
```

Comment:
Bail if any modifiers are set?

Reply:
The only modifier would be clamp to saturate the add. It should work out the same since the clamp bit will be preserved

Reply:
Ack